calibrate.py

```diff
@@ -199,20 +226,39 @@ def parse_args():
                             except IndexError:
                                 complete = True
 
-                if complete == False:
-                    cv2.putText(frame, "Align cameras with callibration board and press spacebar to capture the image", (0, 25), cv2.FONT_HERSHEY_SIMPLEX, 1.0, (0, 255, 0))
+                if complete == False:                    
+                    if framecnt % 60 is 0:
+                        # Find the chess board corners once a second
+                        if find_chessboard(frame):
+                            rightcolor = (0, 255, 0)
+                        else:
+                            rightcolor = (0, 0, 255)
+                    if framecnt % 61 is 0:
```

Comment:
This is a bit hacky, basically it will run once a second, the 60th frame is the left image and the 61st is the right image, this approach is quite brittle and if there was a dropped frame it could break but I didn't see any dropped frames while using  the calibration tool.

Reply:
If the frame would be dropped, I don't think it would appear in the previewout stream - but I may be wrong - @Luxonis-Brandon may know better.

One little thing - the iterators will be desynced in this aproach - every 60th frame to the left (60, 120, 180, 240) and every 61th - (61, 122, 183, 244) etc. 
```suggestion
                    if framecnt % 60 == 1:
```
would solve this (also, it's better to use `==` instead of `is` - this causes the SyntaxWarning on interactive console, like so
```
>>> 4 % 5 is 0
<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
```